lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -90,6 +92,27 @@ class DetailsRenderer {
   }
 
   /**
+   * @param {!DetailsRenderer.LinkDetailsJSON} details
+   * @return {!Element}
+   */
+  _renderLink(details) {
+    const allowedProtocols = ['https:', 'http:'];
+    const url = new URL(details.url);
+    if (!allowedProtocols.includes(url.protocol)) {
```

Comment:
can we do startsWith if we're tryin' to be secure here? otherwise people can just throw http: into their malicious URL ;)

Reply:
it is only checking the `.protocol` property, but yeah startsWith sgtm.. might as well just do `===`

Reply:
wait no.. we're doing `array.includes(string)`.. this uses a `===` under the hood so we're good here.

Reply:
oh whoops yeah I misread as `url.includes(allowedProcotol)` haha, nevermind carryon :)